### DISCUSSION TITLE
Request Maintainer for specific apps

### DISCUSSION THREAD

**Discussion by dnet890** [Homebrew/discussions#1767] [Casks]
Hi
I am just curious that how to request maintainers/somebody to make apps even for beta or dev version to be available on homebrew? because I only find this one (https://docs.brew.sh/New-Maintainer-Checklist) Please advise. Thanks

**Comment by bevanjkay** [comment]
Hi @dtantono, Homebrew does not accept requests for apps to be added by maintainers. However, anybody is able to submit a Pull Request to have an app added or updated, subject to maintainer approval. 

Here is documentation on how to contribute to Homebrew. 
https://github.com/Homebrew/homebrew-cask/blob/master/CONTRIBUTING.md

**Comment by dnet890** [comment]
Ok, so a maintainer in homebrew only plays a role as a mediator in that sense and everyone expect to submit their preferred app individually without any helper. Is that so?

**Reply by bevanjkay** [reply]
Homebrew maintainers review every Pull Request before they are merged, and also assist with triaging issues. The documentation is quite thorough to guide you through submitting a new cask, or an update to an existing one. Even submitting a draft is a good start, and if you do get stuck, you would be able to ask other users or maintainers for help with completing your draft.

**Reply by dnet890** [reply]
Noted and i found that policy is quite different from Microsoft winget or chocolatey. Thank you
